public/js/reader.js

```diff
@@ -2,24 +2,27 @@
 //also handles the thumbnail archive explorer.
 
 function moveSomething(e) {
-
+    if (e.target.tagName === "INPUT") {
+        return;
+    }
 	switch (e.keyCode) {
-		case 37:
-			// left key pressed
+		case 8:   // backspace
+			returnToIndex();
+			break;
+		case 37:  // left arrow
+		case 65:  // a
 			advancePage(-1);
 			break;
-		case 32:
-			// spacebar pressed
+		case 39:  // right arrow
+		case 68:  // d
+			advancePage(1);
+			break;
+		case 32:  // spacebar
 			if ((window.innerHeight + window.scrollY) >= document.body.offsetHeight) {
 				advancePage(1);
```

Comment:
Might as well fix the spacebar issue while we're at it. 👍 
It's a bit cheap to just check readorder and negate the value a second time, but I don't think it's worth refactoring advancePage over.
```suggestion
(localStorage.readorder === 'true') ? advancePage(-1) : advancePage(1);			
```